core/common/src/serializers/TimeZoneSerializers.kt

```diff
@@ -16,6 +15,10 @@ import kotlinx.serialization.encoding.*
  *
  * JSON example: `"Europe/Berlin"`
  */
+@Deprecated(
+    "Serializing TimeZone is discouraged. Please serialize the id String instead.",
```

Comment:
`id String` sounds weird. Maybe you want `id: String`? Or since this is more of a text message, `string id` sounds better IMO